platform/android/MapboxGLAndroidSDKTestApp/src/main/res/layout/activity_marker_view.xml

```diff
@@ -18,6 +18,7 @@
         android:layout_below="@id/toolbar"
         app:center_latitude="38.907192"
         app:center_longitude="-77.036871"
+        app:texture_mode="true"
```

Comment:
I added explicitly a deprecation comment in PolygonActivity ( this should be obvious for end users hence the line is als strikethrough). I removed it from the MarkerViewActivity since it was less obvious.